src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -355,6 +357,12 @@ export function DistributionForm({
     }
   };
 
+  const shouldDisableGiftInput = () => {
```

Comment:
i don't think this needs to be a function

Reply:
updated